inventories/providers/abb/ssc600_vm_standalone_example.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2024, SFL (https://savoirfairelinux.com)
+# SPDX-License-Identifier: Apache-2.0
+
+all:
+  children:
+    VMs:
+      hosts:
+        ssc600-2:
+          description: "vm abb"
+          ansible_host: 192.168.216.147
+          vm_template: "../templates/vm/ssc600.xml.j2"
```

Comment:
does this really work when the inventory file is located in `inventories/providers/abb` ?